nfs/nfsd.go

```diff
@@ -0,0 +1,174 @@
+// /proc/net/rpc/nfsd parsing documented by https://www.svennd.be/nfsd-stats-explained-procnetrpcnfsd/
+package nfs
+
+// rc line: Reply Cache
+type NFSdReplyCache struct {
```

Comment:
Since this package is called `nfs` (maybe it should be `nfsd`), you should omit the prefix for these types: `ReplyCache`, `FileHandles`, etc.

Not sure if `golint` clean is enforced in procfs but I'd run it on this code and update the doc comments.

Reply:
Yea, I started it this way because I wanted to include `nfs` and `nfsd` stats in the same package.

Reply:
If they're very related or share some of the same structures, that seems fine.  If not, I'd just make two separate packages and clarify with doc comments.